### DISCUSSION TITLE
OBS Driver

### DISCUSSION THREAD

**Discussion by shersoni610** [letmaik/pyvirtualcam#50] [Q&A]
Hello,

OBS is a heavyweight software with many great tools with adjustable parameters. Howver, I need
only the virtual camera to send my OpenCV images to the driver. Is it possible to use the Virtual
Cemera device only without downloading OBS and creating the virtual camera by clicking the
button?

Thanks.

**Comment by letmaik** [comment]
In short, no. OBS doesn't offer a stand-alone package of their virtual cameras, which makes sense since they are officially only supposed to be used within OBS. Having said that, both the Windows and macOS variants of the OBS virtual cameras are built as individual components, and it is possible to just install those (but you still need to download the whole package), but keep in mind this is an unsupported and unofficial way to do things. You can find some inspiration in the following two scripts which are used as part of automated testing for this repo:
https://github.com/letmaik/pyvirtualcam/blob/master/.github/scripts/install-macos-obs-virtual-cam.sh
https://github.com/letmaik/pyvirtualcam/blob/master/.github/scripts/install-windows-obs-virtual-cam.ps1
